nx-dev/feature-conf/src/lib/conf-schedule-short.tsx

```diff
@@ -60,6 +60,15 @@ export function ConfScheduleShort(): JSX.Element {
       speakers: ['Jo Hanna Pearce'],
       videoUrl: '',
     },
+    {
+      type: 'event',
+      time: '',
+      title: 'Nx / NxCloud team panel discussion',
+      description:
+        'A Q&A panel with Nx and NxCloud teams ready to answer all your burning questions!',
+      speakers: ['Nx / NxCloud teams', 'Juri Strumpflohner', 'Victor Savkin'],
```

Comment:
yeah just mention "Nx / Nx Cloud teams". Otherwise me and Victor feel like we're not part of that 😜